src/widget/analog-scale/analog-scalepicker.js

```diff
@@ -133,11 +133,15 @@ class AnalogScaleWidget extends RangeWidget {
     get props() {
         const props = this._props;
         props.touch = support.touch;
-        props.min = isNumber( this.element.getAttribute( 'min' ) ) ? this.element.getAttribute( 'min' ) : 0;
-        props.max = isNumber( this.element.getAttribute( 'max' ) ) ? this.element.getAttribute( 'max' ) : 100;
-        props.step = isNumber( this.element.getAttribute( 'step' ) ) ? this.element.getAttribute( 'step' ) : 1; //( props.type === 'decimal' ? 0.1 : 1 );
         props.vertical = !props.appearances.includes( 'horizontal' );
         props.ticks = !props.appearances.includes( 'no-ticks' );
+        props.showScale = props.appearances.includes( 'show-scale' ) && props.vertical && props.ticks;
```

Comment:
Got it. So this is how we restrict this feature to vertical style, with show-scale, and with no-ticks.

Reply:
Hmm, I'm not sure what error should I look into with this line. It looks like that all requirement is fulfilled.
Actually, I become more understanding of what the ticket says by looking into this line.

Reply:
Oh sorry, I didn't leave an error deliberately! :) I was just encouraging a thorough look!